### PR TITLE
Add iPad and Catalyst support for Config quickstart

### PR DESCRIPTION
**Pull request by ncooke3**
This is what the Catalyst app looks like: (The window is resizable. There is a minimum size.)
<details>
<summary>See here</summary>

[
Screen Shot 2021-07-28 at 4 42 25 PM
](url)
</details>

### CODE REVIEW

**Comment by paulb777** [comment]
Please add Catalyst to the CI as well.

File: .github/workflows/config.yml [hunk 1]
```diff
@@ -42,3 +42,21 @@ jobs:
         env:
           SWIFT_SUFFIX: ''
           SAMPLE: Config
+      - name: Build Catalyst
+        run: |
+          cd config
+          ../scripts/install_prereqs/config.sh
+          pod install
+          xcodebuild \
```
Comment (paulb777) [inline .github/workflows/config.yml:50]: We need to add something like the firebase-ios-sdk's build.sh script for managing xcodebuild options, but that can be done separately.

**Review by paulb777 — APPROVED** [review]
Looks great! Thanks!

**Review by paulb777 — APPROVED** [review]
Please update the README to describe how to build for iOS or Catalyst.

**Review by morganchen12 — APPROVED** [review]
LGTM

**Review by paulb777 — APPROVED** [review]

**Pull request merged**